csv_medadmin.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Did you mean to commit this file and csv_patient.json?

Reply:
I did but they should probably go to an examples folder - they are sample csv format files for exporting fhir list as --csv <format file>.

Reply:
I moved the files to examples/csv_format folder.